### DISCUSSION TITLE
New AI-generated icon

### DISCUSSION THREAD

**Discussion by PatriRoman** [Telefonica/mistica-design#1951] [Proposals]
### Goal

Añadir un icono a la librería de iconos de Mística que sea indicativo de "AI-generated".

### Context

Para el CMS necesitamos un icono para usar como indicativo de contenido/cosas generadas por AI. De momento nuestra idea es usar el de MUI ( https://www.figma.com/design/VKl3BiKHO6zeokalZGGjUh/MUIstica?node-id=11033-145245&t=bTiD8vOdiqMmQFHV-1 ) pero lo ideal sería tener uno igual o similar en la librería de Mística, ya que prevemos que con la entrada de la AI será de mucha utilidad.

### Impact

Crear icono nuevo en light, regular y filled.

### Discussion Questions

Entiendo que lo más sencillo es la adaptación del de MUI al estilo de iconos de Mística, ¿o proponéis crear otro distinto?

### Do you have a visual proposal?

https://www.figma.com/design/vRJsHm2p4mmKa8gHr3Y13T/Media-CMS-phase-I?node-id=28-7394&t=u4chRSz5EmORam4R-1

**Comment by andredelramo-hm** [comment]
Buenas Patri, adjunta cuando puedas los 3 pesos del icono (light, regular, filled) y lo añadimos sin problema 🙂

**Comment by yceballost** [comment]
https://github.com/Telefonica/mistica-design/issues/1992
